cmake/CVSLoggerConfig.cmake.in

```diff
@@ -0,0 +1,12 @@
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro)
+
+if(@CVSLOGGER_OPENCV_IMG@)
+  find_dependency(OpenCV)
+endif()
+
+find_dependency(spdlog)
+find_dependency(PkgConfig)
+pkg_check_modules(systemd REQUIRED IMPORTED_TARGET libsystemd)
```

Comment:
This seems to have a potential problem that when the logger is searched for **without** `REQUIRED`, failing to find libsystemd will still fail the configuration.
It's probably not that important, but it's better to forward `REQUIRED` and `QUIET` from the `find_package` call (check `${CMAKE_FIND_PACKAGE_NAME}_FIND_QUIETLY` and `${CMAKE_FIND_PACKAGE_NAME}_FIND_REQUIRED`).